src/js/components/FormField/README.md

```diff
@@ -10,6 +10,7 @@ A single field in a form. FormField wraps an input component with
 import { FormField } from 'grommet';
 <FormField />
 ```
+Documentation for this component: https://v2.grommet.io/formfield
```

Comment:
We can remove everything else from this file besides this line. (See readme files for other components for an example)